src/main/java/dev/suvera/scim2/client/HttpLoggingInterceptor.java

```diff
@@ -25,17 +28,12 @@ public Response intercept(Chain chain) throws IOException {
         Request request = chain.request();
 
         long t1 = System.nanoTime();
-        System.out.println("\nOkHttp [" + purpose + "] " +
-                String.format("Sending %s request %s on %s%n\n%s\nRequestBody:%s",
-                        request.method(), request.url(), chain.connection(), request.headers(), request.body())
-        );
+        log.debugf("\nOkHttp [%s] Sending %s request %s on %s%n\n%s\nRequestBody:%s", purpose, request.method(), request.url(), chain.connection(), request.headers(), request.body());
```

Comment:
Just wondering, do we need to preserve newlines since this isn't going to stdout exclusively anymore?

Reply:
This is going to be a message property on JSON logging let's say. We need to keep some way of separating the information.

Maybe there are better ways of formatting the message, but since these messages are intended to stay on the debug level I think it's ok.

Will get rid of the preceding newling though.